lib/screens/explore_screens/vote_screens/vote/interactor/mappers/remaining_time_state_mapper.dart

```diff
@@ -12,11 +12,11 @@ const int _hourSecond = 60 * 60;
 const int _minuteSecond = 60;
 
 class RemainingTimeStateMapper {
-  VoteState mapResultToState(VoteState currentState, int remainingTime) {
```

Comment:
Note: The remaining time here was a fixed number that never changed - it doesn't need to be passed in here, or set again in the state, since it always remains constant. 
It was actually the end timestamp, just a really confusing name for it.

Cleaned this up.